libs/components/tiles/testing/src/modules/tiles/tile-dashboard-harness.spec.ts

```diff
@@ -28,10 +36,26 @@ describe('Tile dashboard test harness', () => {
         )
       : await loader.getHarness(SkyTileDashboardHarness);
 
-    return { tileDashboardHarness };
+    return { tileDashboardHarness, mediaQueryController };
   }
 
-  it('should get the tile dashboard and a tile by data-sky-id', async () => {
+  it('should get the tile dashboard by data-sky-id', async () => {
+    const { tileDashboardHarness, mediaQueryController } = await setupTest({
+      dataSkyId: 'basic-dashboard',
+    });
+
+    mediaQueryController.setBreakpoint('xs');
+    await expectAsync(tileDashboardHarness.isMultiColumn()).toBeResolvedTo(
+      false,
```

Comment:
can we put the testing of `isMultiColumn` into its own unit test and not in the dataskyid one

Reply:
clarified